src/main/java/com/teamc/fms/entity/BookingType.java

```diff
@@ -0,0 +1,25 @@
+package com.teamc.fms.entity;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import jakarta.validation.constraints.NotNull;
+import lombok.*;
+
+@Entity
+@Table(name = "booking_type")
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class BookingType {
+    @Id
+    private Integer id;
+    @NotNull
+    private String name;
+    @NotNull
+    private String description;
```

Comment:
description column is nullable
![image](https://github.com/web-development-course-way/fms/assets/31379298/5728ba4d-3243-4bd2-8037-6eb3a62b3752)